crates/fluvio-storage/src/segments.rs

```diff
@@ -178,14 +178,6 @@ impl SegmentList {
         self.segments.len()
     }
 
-    #[instrument(skip(self))]
```

Comment:
why was this deleted?

Reply:
I saw that in the output of the failure:
```rust
warning: associated function `get_total_logs_len` is never used
   --> crates/fluvio-storage/src/segments.rs:182:12
    |
182 |     pub fn get_total_logs_len(&self) -> usize {
    |            ^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default
```

https://github.com/infinyon/fluvio/actions/runs/2921183973/attempts/1

Reply:
got it.